client/src/api/Models.ts

```diff
@@ -41,6 +41,12 @@ export const DEFAULT_FFT_PARAMETERS: FFTParams = {
   magnitude_max: 20,
 };
 
+export enum ClientType {
+  API = 'api',
+  LOCAL = 'local',
+  BLOB = 'azure_blob',
+}
+
 export const CLIENT_TYPE_API = 'api';
```

Comment:
why keep these?

Reply:
Will remove them but thought about doing that in another pr

Reply:
It's just a temporary *cough* permament *cough* solution